frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx

```diff
@@ -62,6 +62,10 @@ export const ChartSettingOrderedSimple = ({
   paddingLeft,
   hideOnDisabled = false,
 }: ChartSettingOrderedSimpleProps) => {
+  const processedItems = hideOnDisabled
```

Comment:
nit 

Might be helpful in the future if we could give this const a more specific name.

So instead of `processedItems`, maybe something `orderedAndEnabledItems`.

Might reduce cognition overload in the future I guess.